### PR TITLE
Add Class Search Command, update requirements

### PR DESCRIPTION
**Pull request by ZenKitty**
Adds a `!classes` command that lets users search for class times of a given subject and class number

### CODE REVIEW

**Review by CodeMastr3 — CHANGES_REQUESTED** [review]
I'm not sure if this is necessary refer to pull request #76 where I asked the same question

**Comment by CodeMastr3** [comment]
Will approve after #78 is approved and is mergeable

**Comment by Ramsfield** [comment]
Yup, this looks about right. The dependencies on the bot are ass right now. We seriously should put in some time to get the bot up to date with a supported lib.

**Pull request merged**
